cve_bin_tool/file.py

```diff
@@ -0,0 +1,56 @@
+#!/usr/bin/python3
+
+"""
+This is the customized file class, by doing this
+the tool is able to support other operating systems like Windows
+and MacOS.
+"""
+
+import string
+import sys
+import platform
+import binascii
+
+class File(object):
+
+    def __init__(self, filename = ''):
+        self.filename = filename
+        self.output = ''
+        self.os = platform.system()
+        self.signature = ''
+        if filename == '':
+            pass
+        else:
+            self.signature = self.read_signature().decode('utf-8')
+
+    def read_signature(self):
+        # read signature (first 4 bytes) of the file
+        with open(self.filename, 'rb') as f:
+            byte = f.read(4)
+            return binascii.hexlify(byte)
+
+    def check(self):
+        # make sure class has signature before check
+        if self.signature == '':
+            self.signature = self.read_signature().decode('utf-8')
+        if self.os == 'Linux':
```

Comment:
We want to be able to check all file types on all systems. If I download a zip of `exe`s on a Linux system. I still want to scan them. So just run all the methods.